backend/app/ContentProviders/FourSquare/VenueFactory.php

```diff
@@ -160,7 +160,10 @@ private function setVenueImages($images, Venue $venue)
         foreach($images as $image) {
             $newImage = new VenueImage();
             $newImage->setShotAt(\Carbon\Carbon::createFromTimestamp($image->createdAt)->toCookieString());
-            $newImage->setPhotographer(ucfirst($image->user->firstName) . " " . ucfirst($image->user->lastName));
+            if(isset($image->user->lastName))
+                $newImage->setPhotographer(ucfirst($image->user->firstName) . " " . ucfirst($image->user->lastName));
+            else
```

Comment:
This should be in pr #69

Reply:
It will be, if this gets merged (that's why it's a pr into #69)